sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnSignatures.java

```diff
@@ -120,6 +148,129 @@ private static DoFnSignature parseSignature(Class<? extends DoFn> fnClass) {
               errors.forMethod(DoFn.Teardown.class, teardownMethod), teardownMethod));
     }
 
+    DoFnSignature.GetInitialRestrictionMethod getInitialRestriction = null;
+    ErrorReporter getInitialRestrictionErrors = null;
```

Comment:
Can we move these variable declarations into the if like you did with the others to be consistent?